### PR TITLE
Documentation Edit

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

Changed and added a line about not claiming more than three open issues at a time.

### Type of Change

- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -116,7 +117,7 @@ To link your Codecademy user profile to GitHub:
 
 - Before making a PR, make sure you pushed your changes from a branch other than `main`.
 - Name the new branch after the changes being pushed to the PR.
-- Keep your PRs byte-sized. No more than 3 new entries per PR! And no more than 3 PRs open at a time!
+- Keep your PRs byte-sized. No more than 3 new entries per PR! And do not open more than 3 PRs in a two week period!
```
Comment (KTom101) [inline CONTRIBUTING.md:120]: Would this apply if all their PRs are merged before two weeks are up? I'm thinking for cases of edits which typically are faster to merge.

**Review by KTom101 — COMMENTED** [review]
@SSwiniarski Just one comment on this!

File: CONTRIBUTING.md (hunk 1 above)
Comment (SSwiniarski) [reply CONTRIBUTING.md]: @KTom101 I changed it to not having more than three PRs open in a two week period.

**Review by KTom101 — APPROVED** [review]

**Review by caupolicandiaz — APPROVED** [review]
@SSwiniarski LGTM

**Pull request merged**